README.rst

```diff
@@ -22,43 +23,102 @@ Query Parameters:
 * ?[columnheader1]=[value1]&[columnheader3]=[value4]  (returns results that have both value1 and value2)
 
 
+Requirements
+------------
+
+In order to use autoapi, you'll need:
+
+* Python 3.5
+* npm
+
+Alternatively, if you use Docker for development and/or deployment, you don't
+need anything (except Docker, of course).
+
 
 Quickstart
 ----------
 
-::
+First, let's get dependencies sorted out::
+
+    pip3 install -r requirements.txt
+    npm install
+
+The easiest way to get started with autoapi is by serving CSV files from
+your local filesystem.
+
+Here's a quick way to generate some sample data::
+
+    mkdir data_sources
+    cat << EOF > data_sources/my_sample_data.csv
+    name,color
+    apple,red
+    grapefruit,pink
+    EOF
+
+Now you can add your sample data to a local SQLite database::
+
+    invoke apify "/data_sources/**.*"
+
+Finally, you can start the server with::
 
-    pip install invoke
-    invoke requirements
-    export AWS_ACCESS_KEY_ID="..."
-    export AWS_SECRET_ACCESS_KEY="..."
-    export AUTOAPI_BUCKET="..."
     invoke serve
 
-    curl http://localhost:5000
-    curl http://localhost:5000/sample
-    curl http://localhost:5000/sample/1
-    curl http://localhost:5000/sample?LastName=SMITH
+Now try visiting http://localhost:5000 in your browser, or poke at
+the following URLs from the command-line::
 
-Via docker
-----------
+    curl http://localhost:5000/my_sample_data
+    curl http://localhost:5000/my_sample_data/1
+    curl http://localhost:5000/my_sample_data?color=pink
+
+More details can be found in `instructions.md <https://github.com/18F/autoapi/blob/master/instructions.md>`_.
```

Comment:
Yes, I wrote instructions.md separately partially b/c of a vague feeling that it would be "too long" for the README (whatever that means), but mostly b/c I wanted to work out directions without stomping on what had already been written in the README, and never made it as far as actually merging or meshing them.  They do need it, though.